setup.py

```diff
@@ -317,6 +268,7 @@ def run(self):
         'vispy': [op.join('io', '_data', '*'),
                   op.join('html', 'static', 'js', '*'),
                   op.join('app', 'tests', 'qt-designer.ui'),
+                  op.join('util', 'fonts', 'data', '*.ttf'),
```

Comment:
Do these need to be added to `MANIFEST.in` to be included in the source distribution?